src/fsharp/vs/Symbols.fsi

```diff
@@ -682,6 +682,10 @@ and [<Class>] FSharpMemberOrFunctionOrValue =
 
     member CurriedParameterGroups : IList<IList<FSharpParameter>>
 
+    /// Gets the overloads for the current method
+    /// matchParamterNumber indicates whether to filter the overloads to match the number of parameters in the current symbol
+    member Overloads : bool -> FSharpMemberOrFunctionOrValue list option
```

Comment:
In terms of consistency, shouldn't you return an ```IList<>``` or something like this? See the method above which returns an ```IList<...>```.